public/docs/api-docs.json

```diff
@@ -0,0 +1,23 @@
+{
+  "apiVersion": "0",
+  "swaggerVersion": "1.2",
+  "basePath": "http://localhost:3000/",
```

Comment:
I forgot to mention. Once you deploy the code to the prod machine, you have to run the task to regenerate the docs.

rake swagger:docs